docs: rework wording (#837)
morpho-org/vault-v2: README.md

They hold positions on behalf of the vault.
Adapters are also used to know how much these investments are worth (interest and loss realization).

An [adapter registry](https://github.com/morpho-org/vault-v2/blob/main/src/VaultV2.sol#L89-L97) is used to constrain which adapter a vault can have and add.
This is notably useful when abdicated (see [timelocks](#timelocks)), to ensure that a vault will forever supply into adapters authorized by a given registry.
Morpho Vaults V2 must set and abdicate the [Morpho Registry](https://docs.morpho.org/curate/concepts/adapter-registry/#the-morpho-registry).

The following adapters are currently available:

- [Morpho Market V1 Adapter V2](./src/adapters/MorphoMarketV1AdapterV2.sol).
- [Morpho Vault V1 Adapter](./src/adapters/MorphoVaultV1Adapter.sol).
- Morpho Market V2 Adapter. WIP

### Caps

The funds allocation of the vault is constrained by an id-based caps system.
An id is an abstract identifier for a common risk factor of some positions (a collateral, an oracle, a protocol, etc.).
Allocation on markets with a common id is limited by absolute caps and relative caps.

> Note that relative caps are "soft" because they are not checked on withdrawals, they only constrain new allocations.

Reviewer: not a big fan of the "soft"
```suggestion
Relative caps are allocation-time constraints: they are enforced on new allocations, not on withdrawals.
```

# Vault V2

Vaults V2 enables anyone to create [non-custodial](#non-custodial-guarantees) vaults that allocate assets into different markets.
Depositors of Vault V2 earn from the underlying protocols without having to actively manage their position.
The curation of deposited assets is handled by a set of different roles (owner, curator and allocators).
The [VaultV2Factory](./src/VaultV2Factory.sol) deploys instances of Vaults V2.
All the contracts are immutable.

## Overview

### Adapters

> Vaults allocate assets to underlying protocols via separate contracts called adapters.

Reviewer 2: There's a case to put market here as well.
